packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -1943,7 +1943,17 @@ fn run_experiment(
             v8::V8::initialize_platform(platform);
             v8::V8::initialize();
 
-            let mut isolate = v8::Isolate::new(Default::default());
+            let create_params = v8::Isolate::create_params().heap_limits(
+                init_msg.v8_initial_heap_constraint,
+                init_msg.v8_max_heap_constraint,
+            );
+            let mut isolate = v8::Isolate::new(create_params);
+
+            isolate.add_near_heap_limit_callback(
+                near_heap_limit_callback,
+                // This pointer is not used.
```

Comment:
What is this pointer about?

Reply:
This pointer is passed to the callback, it's the first argument.
This way you can communicate to the callback, but we don't do anything in the callback and don't need additional communication.

Reply:
```suggestion
                // The callback does not need additional data
```